app/templates/methods/magento.logistics.carrier_management.request_shipping_details.json

```diff
@@ -0,0 +1,75 @@
+{
+    "shipment": {
+        "store_id": "{{order.store}}",
+        "order_id": "{{order.id}}",
+        "source_id": "",
```

Comment:
As long as source handling is not supported by the mock (it will be in the future), I would add `"source_id": "XXX"` to have at least something you can work with.